### DISCUSSION TITLE
Existing systems not defined in config.yml will be deleted. Please make regular backups.

### DISCUSSION THREAD

**Discussion by roughnecks** [henrygd/beszel#217] [Support] [answered]
![image](https://github.com/user-attachments/assets/7cdbd2c9-b795-47ef-92ba-adcb33957344)

Is it "may be managed" or "must be managed"?
Do I need to create the config.yml file? I don't have a data directory but a docker volume.

**Comment by henrygd** [comment, marked as answer]
You can ignore this and nothing will change.

It's completely optional and mainly useful for people who want external control via Ansible or the like.

It will only take effect if you've created the `config.yml` file and defined valid systems within it.

**Comment by roughnecks** [comment]
Gotcha, thanks.
